marketplacetests/tests/test_marketplace_make_an_in_app_payment.py

```diff
@@ -11,38 +12,42 @@
 
 class TestMakeInAppPayment(MarketplaceGaiaTestCase):
 
-    APP_NAME = 'Testing In-App-Payments'
-    APP_TITLE = 'In-App-Payments'
-    PIN = '1234'
+    test_data = {
+        'app_name': 'Testing In-App-Payments',
+        'app_title': 'In-App-Payments',
+        'server': 'marketplace-dev.allizom.org',
```

Comment:
I had originally just hardcoded the value in the `set_server` method, as I'm not sure that we'll ever need to change it, but then I noticed a hardcoded value and that looked bad to me. It would make sense that if we did ever need to override it that it would be required for all tests, so finding a way to make this a single value makes sense. Although, maybe just setting a default for now makes the most sense as currently there is no need to override it. I think I'll do that and see what you think of that version.